g1ant-language/refining-your-code.md

```diff
@@ -15,21 +15,21 @@ excel.getvalue row ♥rownumber colname f result ♥cell6
 
 There is the `♥rownumber` variable which takes its value from the `for` loop, so it stays the same in all 6 rows of the procedure. How can you make the other two arguments change incrementally in the same line as the first one? By employing a list variable and another loop.
 
-Let’s start with the `♥cell` variable. Instead of creating a new variable for each cell, you could just create a list variable and add each cell’s content to this list as a value of a subsequent element. The `♥cell` list variable would look like this: `⟦cell1❚cell2❚cell3❚cell4❚cell5❚cell6⟧`. In order to move through the list elements, you would use the good old `foreach` loop or ordinary `for` loop and incremental indexing in the list variable.
+Let’s start with the `♥cells` variable. Instead of creating a new variable for each cell, you could just create a list variable and add each cell’s content to this list as a value of a subsequent element. The resulting `♥cells` list variable in this case would look like this: `cell1❚cell2❚cell3❚cell4❚cell5❚cell6`. In order to move through the list elements, you would use the good old `foreach` loop or ordinary `for` loop and incremental indexing in the list variable.
```

Comment:
you have changed your code but this list remained, so please update this description :)

Reply:
so maybe add something like "but you can create an empty list instead" and then something like "you can do this using list.create command"

Reply:
"The **resulting** `♥cells` list variable in this case **would** look like this: "

key words: **resulting** and **would**

Reply:
you don't explain how to use list.create command